RubiksCubeSolver.h

```diff
@@ -240,9 +249,68 @@ int RubiksCubeSolver::bidirectionalBFS() {
             break;
         }
     }
+    
+    
+    fwdPath = retraceHalfPath(cubesSeenFwd, currentCube.toString());
+    //std::cout << fwdPath << std::endl;
+    bkwdPath = retraceHalfPath(cubesSeenBkwd, currentCube.toString());
+    //std::cout << bkwdPath << std::endl;
+    
     return ans;
 }
 
+std::string RubiksCubeSolver::retraceHalfPath(Trie cubesSeen, std::string currentCubeString) {
+    int currentCubeIndex = cubesSeen.getCubeIndex(currentCubeString);
+    
+    std::string halfPath = "";
+    
+    while (currentCubeIndex != -1) {
+        halfPath.append(cubesSeen.getLastMove(currentCubeIndex));
+        //std::cout << halfPath << " " << currentCubeIndex << std::endl;
+        
+        currentCubeIndex = cubesSeen.getParentIndex(currentCubeIndex);
+    }
+    
+    return halfPath;
+}
+
+std::string RubiksCubeSolver::getPath() {
```

Comment:
this function is a bit too complex, refer to getLastMove comment in Trie structure below